tests/test_homepage.py

```diff
@@ -16,32 +16,27 @@ class TestHomepage(BaseTest):
 
     @pytest.mark.moztrap([3385, 3386])
     @pytest.mark.nondestructive
-    def test_that_user_can_login_and_logout(self, mozwebqa):
+    def test_that_user_can_login_and_logout(self, mozwebqa, variables):
```

Comment:
Do you think it's worth adding an `existing_user` fixture, so we don't have to interact with `variables` inside the test? Maybe this is the only test that would use it and that's why it's not worthwhile.